scripts/auto-rebase/rebase.sh

```diff
@@ -1118,6 +1119,44 @@ EOF
     done  # for goarch
 }
 
+update_kubeproxy_images() {
```

Comment:
Do you know what's the difference between `kube-proxy` and `kube-rbac-proxy`? Can they be used interchangeably?